src/bio/bio_internal.h

```diff
@@ -539,6 +538,7 @@ extern struct bio_faulty_criteria	glb_criteria;
 /* bio_xstream.c */
 extern bool		bio_scm_rdma;
 extern bool		bio_spdk_inited;
+extern bool                             bio_vmd_enabled;
```

Comment:
[style] bio_vmd_enabled should be aligned with above, looks an extra tab here. ^_^

Reply:
This is enforced by clang-format and I don't want to fight with the linter.